### ISSUE TITLE
homebrew/php/php70 7.0.14_7 flagged as virus

### ISSUE THREAD

**Issue by leewillis77** [Homebrew/homebrew-php#3837]
## Description

Trying to update php70, but its getting flagged as containing a virus by AVG:

https://www.dropbox.com/s/wwe9oyr27ovi8pc/Screenshot%202016-12-12%2010.51.29.png?dl=0

May well be a false positive - but thought it worth reporting.

**Comment by leewillis77** [comment]
I also tried to install php71, but that also hits issues:

https://www.dropbox.com/s/0katmplx7tnj9pa/Screenshot%202016-12-12%2011.08.02.png?dl=0

**Comment by leewillis77** [comment]
AVG also complains about the raw source from PHP.net - so reported upstream.

**Comment by javian** [comment]
@leewillis77 Thanks for looking into this.

**Comment by alcaeus** [comment]
@leewillis77 I couldn't find an issue in the PHP bug tracker - did you find out whether it was a false positive?

**Comment by javian** [comment]
I think he might have reported it to AVG and not PHP.

**Comment by leewillis77** [comment]
I reported it to PHP (https://bugs.php.net/bug.php?id=73722 - although it's private). 

The response there was:

"Thanks for the report. The AVG antivirus is not listed on virustotal.com, but other dozens of antiviruses are green. Moreover - the mentioned file is of no use on Mac, it belongs to the build system on Windows. This looks like false positive, which happen to be reported from time to time, so I would suggest you to report the false positive to the antivirus manufacturer."

I reported it to AVG, but have had no response.
